Outreachy Dec 2021 to Mar 2022.md

```diff
@@ -0,0 +1,39 @@
+Oppia is excited to be participating in [Outreachy](https://www.outreachy.org/)'s December 2021 to March 2022 round! Outreachy is a diversity initiative that provides paid, remote internships to people subject to systemic bias and impacted by under-representation in the technical industry where they are living.
```

Comment:
We try to avoid spaces in file names, so how about `Outreachy-Dec-2021-to-Mar-2022.md` instead?